### DISCUSSION TITLE
How to open ports for torrent clients? Seeding problems

### DISCUSSION THREAD

**Discussion by gado4525** [bubuntux/nordlynx#143] [Q&A]
Hello, how do I open listening port for qbittorrent and transmission? I've tried adding the port to the docker-compose.yml along with the clients port but this doesn't seem to open it.  Not having port open is causing me upload issues and problems on private trackers

**Comment by Sparklingx** [comment]
I also want to know

**Comment by Johannes-m-TK** [comment]
without a docker-compose.yml, there is not much to say
